templates/configure_user.j2

```diff
@@ -0,0 +1,16 @@
+{% for user in users %}
+{% if user.state is defined and user.state == 'absent' %}
+delete system login user {{ user.name }}
+{% else %}
+{% if user.role is defined %}
+set system login user {{ user.name }} class {{ user.role }}
+{% endif %}
+{% if user.full_name is defined %}
+set system login user {{ user.name }} full-name {{ user.full_name }}
+{% endif %}
+{% if user.sshkey is defined %}
+{% set key_type = user.sshkey.split(' ')[0] %}
+set system login user {{ user.name }} authentication {{ key_type }} "{{ user.sshkey }}"
```

Comment:
Can setting encrypted password be added as part of this PR. Refer https://github.com/ansible/ansible/pull/44660/files

Reply:
ACK. Added this.